official/nlp/data/classifier_data_lib.py

```diff
@@ -1277,12 +1277,7 @@ def convert_single_example(ex_index, example, label_list, max_seq_length,
   return feature
 
 class AXgProcessor(DataProcessor):
-  """Processor for the AX dataset (GLUE diagnostics dataset)."""
-
-  def get_dev_examples(self, data_dir):
-    """See base class."""
-    return self._create_examples(
-        self._read_jsonl(os.path.join(data_dir, "dev.jsonl")), "dev")
+  """Processor for the AXg dataset (GLUE diagnostics dataset)."""
 
   def get_test_examples(self, data_dir):
     """See base class."""
```

Comment:
Because the name of the downloaded file is "AX-g.jsonl", is it better to use "AX-g.jsonl" as filename here?

self._read_jsonl(os.path.join(data_dir, "test.jsonl")), "test")